tests/files/BlockchainTests/bcUncleTest.json

```diff
@@ -4826,23 +4826,23 @@
         "lastblockhash" : "eed1b4da708283370856fc76352d68f36d9766b7f366da372e2992ced9a1f663",
         "postState" : {
             "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
-                "balance" : "0x14",
+                "balance" : "0x1e",
```

Comment:
So it actually turns out cpp is not checking the values in the postState either, they are just printed when filling tests for debugging. And Go only checks postState accounts if they are also in the pre state. However, it shouldn't matter because both verify the state root of every block, which would be different if there was an actual mismatch in calculated accounts.

Closing this PR and opening a new since digging into the test wrappers resulted in a few more changes.